components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/WorkflowManagementServiceImpl.java

```diff
@@ -169,14 +208,25 @@ public WorkflowEvent getEvent(String id) {
                 templates.add(template);
             }
         }
+        for (WorkflowListener workflowListener : workflowListenerList) {
+            workflowListener.doPostListTemplates(templates);
+        }
+
+
```

Comment:
Thanks for pointing this, fixed with https://github.com/wso2/carbon-identity/pull/1675